tests/unit/stateful-test.js

```diff
@@ -1,12 +1,244 @@
 import Ember from 'ember';
 import StatefulMixin from 'ember-stateful/mixins/stateful';
+import ERRORS from 'ember-stateful/errors';
 import { module, test } from 'qunit';
 
 module('Unit | Mixin | stateful');
 
 // Replace this with your real tests.
 test('it works', function(assert) {
-  let StatefulObject = Ember.Object.extend(StatefulMixin);
+  let StatefulObject = Ember.Object.extend(StatefulMixin, {
+    actions: {}
+  });
   let subject = StatefulObject.create();
   assert.ok(subject);
 });
+
+test('transitions to default state', function(assert) {
+  let StatefulObject = Ember.Object.extend(StatefulMixin, {
+    actions: {
+      A: {}
+    }
+  });
+  let subject = StatefulObject.create();
+  assert.ok(subject.get('state.A'))
+});
+
+test('transitions to default state specified explicitly', function(assert) {
+  let StatefulObject = Ember.Object.extend(StatefulMixin, {
+    actions: {
+      A: {},
+      B: {
+        _default: true,
+      }
+    }
+  });
+  let subject = StatefulObject.create();
+  assert.ok(subject.get('state.B'))
+});
+
+test('throws if default substate not specified', function(assert) {
```

Comment:
better: 'throws if the default state is neither obvious nor specified explicitly'

Reply:
ok